### PR TITLE
solution

### PR DESCRIPTION
**Pull request by mykola-koval**
[DEMO LINK](https://mykola-koval.github.io/react_person/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -25,22 +25,9 @@ const alex = {
...
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person person={misha} />
+    <Person person={olya} />
+    <Person person={alex} />
   </div>
```
Comment (aleksandra-mileant) [inline src/App.js:31]: забыл о git add . ?)

Comment (mykola-koval) [reply src/App.js:31]: ух ты, точно, спасибо большое!)

> забыл о git add . ?)

**Review by Sesile26 — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -25,22 +25,9 @@ const alex = {
...
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person person={misha} />
+    <Person person={olya} />
+    <Person person={alex} />
```
Comment (Elena-brv) [inline src/App.js:30]:
```suggestion
    <Person {...misha} />
    <Person {...olya} />
    <Person {...alex} />
```

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerType = sex === `m` ? `wife's` : `husband's`;
+  const partner = isMarried
+    ? `My ${partnerType} name is ${partnerName}`
+    : `I am not married`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && `I am ${age}`}</p>
```
Comment (Elena-brv) [inline src/components/Person/Person.jsx:15]: you should use conditional rendering for the whole p element, it's no need to have an empty p

**Review by Elena-brv — CHANGES_REQUESTED** [review]

**Review by Elena-brv — APPROVED** [review]

**Pull request closed without merging**